slither.config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "filter_paths": "(contracts/mock/|contracts/ext_libs/|node_modules)"
```

Comment:
I suggest we exclude detectors:
- `uninitialized-local` since we intentionally never initialize default values.
- `timestamp` since we want comparison agains external timestamp and miner's potentioal manipulation has low to no impact
- `assembly` since we use it as little as possible, but when we do, it's intentional (and it's not just some random copy paste). And everything that this detector does it just says "don't use assembly, it's bad", but does not point out actual problems
-

Reply:
I’ve excluded the detectors you mentioned